EdFi.Ods.Utilities.Migration/Scripts/PgSql/02Upgrade/v51_to_v52/01 Structure/1010-AddNewTablesForV52.sql

```diff
@@ -0,0 +1,245 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE edfi.AssessmentScoreRangeLearningStandard (
+    AssessmentIdentifier                  VARCHAR (60)    NOT NULL,
+    Namespace                             VARCHAR (255)   NOT NULL,
+    ScoreRangeId                          VARCHAR (60)    NOT NULL,
+    AssessmentReportingMethodDescriptorId INT              NULL,
+    MinimumScore                          VARCHAR (35)    NOT NULL,
+    MaximumScore                          VARCHAR (35)    NOT NULL,
+    IdentificationCode                    VARCHAR (60)    NULL,
+    Discriminator                         VARCHAR (128)   NULL,
+    CreateDate                            TIMESTAMP   DEFAULT current_timestamp NOT NULL ,
+    LastModifiedDate                      TIMESTAMP   DEFAULT current_timestamp NOT NULL ,
+    Id                                    UUID  DEFAULT gen_random_uuid() NOT NULL,
```

Comment:
In ODS for v5.2 from the package, this line seems like it should end as `UUID  DEFAULT public.gen_random_uuid() NOT NULL,`

Reply:
fixed it